packages/grafana-data/src/transformations/transformers/labelsToFields.test.ts

```diff
@@ -43,38 +43,6 @@ describe('Labels as Columns', () => {
     expect(result[0].fields).toEqual(expected);
   });
 
```

Comment:
Are we not supporting the removed test case any longer?

Reply:
Ah ok I see now, that we don't. So this transformer will have the same issues as before `getFramesWithOnlyValueFields` function was introduced with the long column names?

Reply:
Never mind, I thought we had revert the whole field naming logic.

Reply:
yea, this logic is no longer needed :) as now time series always have the field name "Value" and no longer the same as the frame name, this removed test was for the logic that handled frame.name === field.name and that logic is no longer needed